mx.truffleruby/mx_truffleruby.py

```diff
@@ -19,7 +19,7 @@
     import mx_truffleruby_benchmark
 
 _suite = mx.suite('truffleruby')
-rubyDists = ['TRUFFLERUBY', 'TRUFFLERUBY-TEST']
+root = _suite.dir
```

Comment:
Could we look to see if the `sulong` suite is available, and if so set `SULONG_HOME` to it automatically?